Boards/arduino_mega.board.json

```diff
@@ -48,7 +48,8 @@
     "MaxOutputs": 40,
     "MaxServos": 10,
     "MaxShifters": 4,
-    "MaxSteppers": 10
+    "MaxSteppers": 10,
+    "MaxDigInputMuxes": 2,
```

Comment:
Please let's stick to long names.

MaxInputMultiplexer

I think we don't need "Dig" in the name. Would we need a specific multiplexer if it supported analog?

Reply:
The UI would be the same, but would just have to distinguish between an analog or digital input - definitely in the firmware. I guess a different object is not required, an attribute would be sufficient.